docs/UserGuide.md

```diff
@@ -202,37 +202,43 @@ Examples:
 
 Updates the amount of tuition fee paid by the specified student after a lesson.
 
-Format: `pay INDEX hr/HOURS_PAID`
+**Format:** `pay INDEX hr/HOURS_PAID`
 
-Example:
+**Example:**
 * `pay 1 hr/2.5` updates the tuition amount paid by the 1st student in the address book.
-  ![payResult.png](images/payResult.png)
+  
+**Output:**
+![payResult.png](images/payResult.png)
 
 <box type="important" header="##### Constraints">
 
-1. The index refers to the index number shown in the displayed student list.
-2. The index **must be a positive integer** 1, 2, 3, …​
-3. Hours paid field should be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
+* The **INDEX** refers to the index number shown in the displayed student list.
+* The **INDEX must be a positive integer** 1, 2, 3, …​
+* **HOURS_PAID** should be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
 
```

Comment:
Might be good to add in a tip for using edit to change the paid amount if made a mistake. Similar to the tip for owe command below.